src/components/structures/login/Login.js

```diff
@@ -214,7 +214,9 @@ module.exports = React.createClass({
         }).done();
     },
 
-    _onLoginAsGuestClick: function() {
+    _onLoginAsGuestClick: function(ev) {
+        ev.preventDefault();
```

Comment:
If `stopPropagation` is needed for the other cases, should it be added here?

Reply:
oops, I probably just forgot here.